apiserver/src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -3640,17 +3810,13 @@ public void getLatestProjectWithAclEnabledNoAccessTest() {
                 .request()
                 .header(X_API_KEY, apiKey)
                 .get(Response.class);
-        Assert.assertEquals(403, response.getStatus(), 0);
+        Assert.assertEquals(404, response.getStatus(), 0);
```

Comment:
Similar to above, inaccessible projects are filtered out on DB-level hence the response now indicates a "not found".